akvo/rsr/models/budget_item.py

```diff
@@ -119,11 +122,20 @@ def get_label(self):
         else:
             return self.__unicode__()
 
+    def get_currency(self):
```

Comment:
Could be rewritten to:
`return self.currency if self.currency else self.project.currency`